### PR TITLE
Modify rule S6388: Remove diff-id on compliant example without non-compliant equivalent

### PR DESCRIPTION
**Pull request by rudy-regazzoni-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

[Related issue](https://github.com/SonarSource/rspec/pull/1925#pullrequestreview-1624869461)

### CODE REVIEW

File: rules/S6388/azureresourcemanager/examples/Microsoft.Compute_virtualMachines.noncompliant.adoc [hunk 1]
```diff
@@ -152,4 +152,4 @@ resource myName 'Microsoft.Compute/virtualMachines@2022-11-01' = {
     }
   }
 }
-----
\ No newline at end of file
+----
```
Comment (jonas-wielage-sonarsource) [inline rules/S6388/azureresourcemanager/examples/Microsoft.Compute_virtualMachines.noncompliant.adoc:155]: Not sure what is happening here.

**Review by jonas-wielage-sonarsource — APPROVED** [review]
LGTM

File: rules/S6388/azureresourcemanager/examples/Microsoft.Compute_virtualMachines.noncompliant.adoc (hunk 1 above)
Comment (rudy-regazzoni-sonarsource) [reply rules/S6388/azureresourcemanager/examples/Microsoft.Compute_virtualMachines.noncompliant.adoc:155]: Same here :sweat_smile:

**Comment by marco-antognini-sonarsource** [comment]
@rudy-regazzoni-sonarsource  It did. Thanks for the fast response time!

**Pull request merged**